Loop/Extensions/UIAlertController.swift

```diff
@@ -72,35 +72,35 @@ extension UIAlertController {
     /// Initializes an action sheet-styled controller for selecting a CGMManager
     ///
     /// - Parameters:
-    ///   - cgmManagers: An array of CGMManager-conforming types
+    ///   - cgmManagers: An array of CGMManagers
     ///   - pumpManager: A PumpManager/CGMManager combo instance
     ///   - selectionHandler: A closure to execute when either a new CGMManager or the current PumpManager is selected
-    ///   - cgmManager: The selected CGMManager type
-    ///   - pumpManager: The selected PumpManager instance
-    internal convenience init(cgmManagers: [CGMManagerUI.Type], pumpManager: CGMManager?, selectionHandler: @escaping (_ cgmManager: CGMManagerUI.Type?, _ pumpManager: CGMManager?) -> Void) {
+    ///   - cgmManager: The selected CGMManager
+    ///   - pumpManager: The selected PumpManager
+    internal convenience init(cgmManagers: [AvailableDevice], pumpManager: CGMManager?, selectionHandler: @escaping (_ identifier: String, _ pumpManager: CGMManager?) -> Void) {
```

Comment:
Why the switch to `AvailableDevice`?

Reply:
This allows for both static CGMManagers and those loaded by the newer plugin architecture.